packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -444,8 +447,9 @@ class _GlowingOverscrollIndicatorPainter extends CustomPainter {
     this.leadingController,
     this.trailingController,
     this.axisDirection,
+    repaint,
```

Comment:
nit: this argument needs a type (i expect the analyzer will complain about this)